pkg/project/list.go

```diff
@@ -0,0 +1,43 @@
+package project
+
+var (
+	commonProjectList = []string{
+		"cluster-operator",
+		"draughtsman",
+	}
+
+	providerProjectList = map[string][]string{
+		"aws": {
+			"aws-app-collection",
+			"aws-operator",
+		},
+		"kvm": {
+			"kvm-app-collection",
+			"kvm-operator",
+		},
+		"azure": {
+			"azure-app-collection",
+			"azure-operator",
+		},
+	}
+
+	perInstallationProjectLists = map[string][]string{
+		"gaia": {"app-operator"},
```

Comment:
Is the thiccc app-operator deployed to all test CPs? Doesn't look like this change is in master?

Reply:
This is in master already. https://github.com/giantswarm/draughtsman/blob/master/service/eventer/projects.go
I think it's okay? WDYT?

Reply:
OK cool. Yes I think its OK. Thanks for confirming.